docs/resources/aws_connection.md

```diff
@@ -0,0 +1,50 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "clumio_aws_connection Resource - terraform-provider-clumio"
+subcategory: ""
+description: |-
+  Clumio AWS Connection Resource used to connect AWS accounts to Clumio.
+---
+
+# clumio_aws_connection (Resource)
+
+Clumio AWS Connection Resource used to connect AWS accounts to Clumio.
+
+## Example Usage
+
+```terraform
+resource "clumio_aws_connection" "example" {
+  account_native_id           = "aws_account_id"
+  aws_region                  = "aws_region"
+  description                 = "description"
+  protect_asset_types_enabled = ["EBS", "RDS", "DynamoDB", "EC2MSSQL", "S3"]
+  services_enabled            = ["discover", "protect"]
+  organizational_unit_id      = "organizational_unit_id"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- **account_native_id** (String) AWS Account Id to connect to Clumio.
+- **aws_region** (String) AWS Region of account.
+
+### Optional
+
+- **description** (String) Clumio AWS Connection Description.
+- **organizational_unit_id** (String) Clumio Organizational Unit Id.
+- **protect_asset_types_enabled** (List of String) The asset types enabled for protect. This is only populated if protect is enabled. Valid values are any of [EBS, RDS, DynamoDB, EC2MSSQL, S3].
+- **services_enabled** (List of String) The services to be enabled for this configuration. Valid values are [discover], [discover, protect]. This is only set when the registration is created, the enabled services are obtained directly from the installed template after that.
+
+### Read-Only
+
+- **clumio_aws_account_id** (String) Clumio AWS AccountId.
+- **clumio_aws_region** (String) Clumio AWS Region.
+- **connection_status** (String) The status of the connection. Possible values include connecting, connected and unlinked.
+- **id** (String) Clumio AWS Connection Id.
+- **namespace** (String) K8S Namespace.
+- **token** (String) The 36-character Clumio AWS integration ID token.
```

Comment:
`Clumio AWS integration` -> `Clumio connection integration`?
I say this because isn't the token just so that we can find the associated Clumio connection after provisioning of all AWS resources finishes and the callback is made?

Reply:
I took this from the REST API documentation. The full comment in the REST API Documentation for token is "The 36-character Clumio AWS integration ID token used to identify the installation of the CloudFormation template on the account. This value will be pasted into the ClumioToken field when creating the CloudFormation stack." But since it talks about cloudformation, removed that part and just kept first part.

Reply:
Ah in that case I think the latter part if valuable because it does help to identify the installation on the customer's account. Then perhaps keep the whole phrase and write `...  installation of the Terraform template ...`?